lib/util.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+// parseUri 1.2.2
```

Comment:
I think it's better to leave this code totally unmodified. Just temporarily disable any eslint checks that are failing on it -- or disable linting entirely for this section.